packages/shared/src/kebab/kebab.tsx

```diff
@@ -294,7 +296,7 @@ export const Kebab: React.FC<KebabProps> & KebabStaticProperties = ({
       }
     >
       <Dropdown
-        data-test="kebab-button"
+        data-test={dataTestId ?? 'kebab-button'}
```

Comment:
Nullish coalescing operator only checks `null` or `undefined`, but `dataTestId` could also be an empty string or similar falsy value:
```suggestion
        data-test={dataTestId || 'kebab-button'}
```

Reply:
is someone wants to purposefully disable it by passing an empty string we should allow it.

Reply:
What is the purpose of passing a `data-test` empty value? It doesn't make any sense to me.